src/Cli/Exporter.cs

```diff
@@ -80,6 +80,10 @@ public static bool Export(ExportOptions options, ILogger logger, FileSystemRunti
                     logger.LogError("Failed to export GraphQL schema.");
                 }
             }
+            else
+            {
+                logger.LogError("Exporting GraphQL schema is not enabled. You need to pass --graphql.");
```

Comment:
why do we need to pass the `--graphql` option if export is by default always to export the GraphQL schema as per the description? Shouldnt remove the `if` check altogether?

Reply:
This is existing code, so I didn't want to modify it; I only added the error message. Passing this flag has its own benefits, as it ensures that customers have the right expectations when running this command, given that we support both REST and GraphQL APIs.